dot/network/service.go

```diff
@@ -238,10 +240,37 @@ func (s *Service) Start() error {
 
 	logger.Info("started network service", "supported protocols", s.host.protocols())
 
+	if s.cfg.PublishMetrics {
```

Comment:
maybe I missed it but I don't see this being set anywhere? eg in `dot/services.go` `createNetworkService`?

Reply:
Thanks for pointing it. I missed it while refactoring the metrics package.